### ISSUE TITLE
Cho Ren Sha 68K Music Stereo Channels Reversed

### ISSUE THREAD

**Issue by AktaneDev** [MiSTer-devel/X68000_MiSTer#28]
Comparing music of Cho Ren Sha to windows version and to official soundtrack shows that the music channels are reversed. Not sure about the sound effects themselves.

**Comment by Jlmbles** [comment]
I don't know if anyone else has noticed but there is also a bit of audio popping. Which seems to get worse the longer it is running?

**Comment by AktaneDev** [comment]
Yeah agreed. But I don't think it gets worse I think it's just there haha. Autofire seems to possibly make it a bit worse... But it's just there the whole time.

**Comment by odiaboeeu** [comment]
Could you test it with the core that was released?

**Comment by AktaneDev** [comment]
@odiaboeeu

the stereo music is STILL backwards, left to right.. unresolved

the audio pops are gone! I didnt play the whole game but it seems resolved - yay!

there is another issue introduced now, the bomb creates the background to flash white in a strange way - I will create a new ticket

**Comment by odiaboeeu** [comment]
Regarding the music, the left and right channels appear to be inverted. Have you tested it on the XM6g emulator to see how it behaves there?

**Comment by AktaneDev** [comment]
I have not - but I have tested against the windows release of the same game, and also the OST itself which is a GIMIC playback of the original sharp code

**Comment by AktaneDev** [comment]
https://youtu.be/nlZHVQbBMsA @odiaboeeu here is the XM6g - it seems correct here

**Comment by odiaboeeu** [comment]
I tested a small fix that swaps only the OPM/FM L/R assignment before the final mix.

This fixes the reversed stereo pan in Cho Ren Sha 68K music test compared against XM6G. I also tested Akumajou Dracula, where the OPM stereo placement is very noticeable, and the patched MiSTer output now matches XM6G.

ADPCM/PCM routing is unchanged.

[X68000_20260613.zip](https://github.com/user-attachments/files/28907558/X68000_20260613.zip)

**Comment by odiaboeeu** [comment]
This issue has been resolved in the core version released today. If you could test, confirm, and then close this issue, it would be appreciated.

**Comment by AktaneDev** [comment]
HELL YES THIS IS FIXED LETS GOOOO!!!
